modules/installation-initializing.adoc

```diff
@@ -290,7 +290,7 @@ For production {product-title} clusters on which you want to perform installatio
 +
 [source,terminal]
 ----
-https://<engine-fqdn>/ovirt-engine/api <1>
+https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api <1>
```

Comment:
The variable didn't work as it is in a code block, I think. Is this what you expected:
`https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api `

And the call-out has `<engine-fqdn>`.

Reply:
Fixed. Changed back to engine-fqdn.